foundry.toml

```diff
@@ -1,14 +1,26 @@
 [profile.default]
-src = 'contracts'
-out = 'out'
-libs = ['node_modules', 'lib']
-solc_version = '0.8.23'
+src = "contracts"
```

Comment:
If this is a (almost) pure forge project, I'd keep the configuration file as small as possible, removing whatever can follow the default forge value.
Like src and out

Reply:
This is planned, but I didn't want to pollute this PR.
Also, I wanted to ditch the `prettier` packages and use `forge fmt`.
However, their formatting differs every so slightly.
Even so, that would create some noise in the diff.
So, both moving `contracts` to `src` and removing `prettier` is adjourned to future PRs.